lib/msal-common/src/client/AuthorizationCodeClient.ts

```diff
@@ -200,8 +203,13 @@ export class AuthorizationCodeClient extends BaseClient {
         // Deserialize hash fragment response parameters.
         const hashUrlString = new UrlString(hashFragment);
         // Deserialize hash fragment response parameters.
-        const serverParams: ServerAuthorizationCodeResponse =
-            UrlString.getDeserializedHash(hashUrlString.getHash());
+        let serverParams : ServerAuthorizationCodeResponse;
```

Comment:
This can also be factored out into a method called getDeserializedCodeResponse or something like that.